R/spline_helpers.R

```diff
@@ -46,6 +46,14 @@ spline2_create <- function(x, nm = "pred", .fn = "bSpline", df = 3, complete_set
 
 spline_msg <- function(x) {
```

Comment:
For next week: pass the upstream caller env to avoid "Caused by error in `spline_msg()`:"